libraries/OPAMP/examples/start_opamp/start_opamp.ino

```diff
@@ -0,0 +1,20 @@
+#include <OPAMP.h>
+
+void setup () {
+  Serial.begin(9600);
+  delay(2000);  // serial monitor delay
+  // activate OPAMP
+  if (!OPAMP.begin(OPAMP_SPEED_HIGHSPEED)) {
+    Serial.println("Failed to start OPAMP!");
+  }
+  uint8_t status = OPAMP.getStatus();

```

Comment:
```suggestion
  uint8_t const status = OPAMP.getStatus();
```

Reply:
Obsoleted by creation of beter `bool isRunning()`, with adopted sketch code in ef3472ac6527c40fea48a6a1d2d045460ba748de